assets/dev/scss/frontend/widgets/playing-cards.scss

```diff
@@ -0,0 +1,125 @@
+$playing-card-cover-bg-color: #e2e2e2;
+$transition-fast: 0.3s ease;
+$transition-normal: 0.6s;
+
+:root {
+	--e-card-bg-color: #fff;
+	--e-card-border-size: 1px;
+	--e-card-border-color: #000;
+	--e-card-border-radius: 4px;
+	--e-card-padding: 4px --e-card-suite-size: 20px;
+	--e-card-suite-size: 20px;
+	--e-card-suite-color: #000;
+	--e-card-number-color: #000;
+}
+
+.elementor-widget-playing-cards {
```

Comment:
Maybe?

```suggestion
.elementor-widget-playing-cards {
	--e-card-bg-color: #fff;
	--e-card-border-size: 1px;
	--e-card-border-color: #000;
	--e-card-border-radius: 4px;
	--e-card-padding: 4px --e-card-suite-size: 20px;
	--e-card-suite-size: 20px;
	--e-card-suite-color: #000;
	--e-card-number-color: #000;
```

Reply:
I like current implementation, do you have some reason to do it like this?

Reply:
Okay

Reply:
fixed